.theia/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  "version": "0.2.0",
```

Comment:
Hmmm. Are these .theia/launch.json files needed? I don't see them in other peoples repo. 

I'd recommend removing them if possible. Try to keep local workspace files out of the git repository. This helps:
 * Keep a cleaner repository
 * Keep personal dev settings, well, personal